webapi/src/main/scala/org/knora/webapi/messages/v2/responder/ontologymessages/KnoraApiV2WithValueObjects.scala

```diff
@@ -171,1436 +144,490 @@ object KnoraApiV2WithValueObjects {
                 objectsWithLang = Map(
-                    LanguageCodes.DE -> "ist Teil von",
-                    LanguageCodes.EN -> "is part of",
-                    LanguageCodes.FR -> "fait partie de",
-                    LanguageCodes.IT -> "fa parte di"
+                    LanguageCodes.EN -> "Subject type"
                 )
             ),
             makePredicate(
                 predicateIri = OntologyConstants.Rdfs.Comment,
                 objectsWithLang = Map(
-                    LanguageCodes.EN -> "Indicates that this resource is part of another resource"
+                    LanguageCodes.EN -> "Specifies the required type of the subjects of a property"
                 )
             )
         )
     )
 
-    val IsPartOfValue: ReadPropertyInfoV2 = makeProperty(
-        propertyIri = OntologyConstants.KnoraApiV2WithValueObjects.IsPartOfValue,
-        isResourceProp = true,
-        propertyType = OntologyConstants.Owl.ObjectProperty,
-        subjectType = Some(OntologyConstants.KnoraApiV2WithValueObjects.Resource),
-        objectType = Some(OntologyConstants.KnoraApiV2WithValueObjects.LinkValue),
-        subPropertyOf = Set(OntologyConstants.KnoraApiV2WithValueObjects.HasLinkToValue),
-        isEditable = true,
-        isLinkValueProp = true,
+    private val ObjectType: ReadPropertyInfoV2 = makeProperty(
+        propertyIri = OntologyConstants.KnoraApiV2WithValueObjects.ObjectType,
+        propertyType = OntologyConstants.Rdf.Property,
         predicates = Seq(
             makePredicate(
                 predicateIri = OntologyConstants.Rdfs.Label,
                 objectsWithLang = Map(
-                    LanguageCodes.DE -> "ist Teil von",
-                    LanguageCodes.EN -> "is part of",
-                    LanguageCodes.FR -> "fait partie de",
-                    LanguageCodes.IT -> "fa parte di"
+                    LanguageCodes.EN -> "Object type"
                 )
             ),
             makePredicate(
                 predicateIri = OntologyConstants.Rdfs.Comment,
                 objectsWithLang = Map(
-                    LanguageCodes.EN -> "Indicates that this resource is part of another resource"
+                    LanguageCodes.EN -> "Specifies the required type of the objects of a property"
                 )
             )
         )
+
     )
 
-    val HasIncomingLinks: ReadPropertyInfoV2 = makeProperty(
-        propertyIri = OntologyConstants.KnoraApiV2WithValueObjects.HasIncomingLinks,
-        isResourceProp = true,
-        propertyType = OntologyConstants.Owl.ObjectProperty,
-        subjectType = Some(OntologyConstants.KnoraApiV2WithValueObjects.Resource),
-        objectType = Some(OntologyConstants.KnoraApiV2WithValueObjects.LinkValue),
-        subPropertyOf = Set(OntologyConstants.KnoraApiV2WithValueObjects.HasLinkToValue),
-        isEditable = true,
-        isLinkValueProp = true,
+    private val TextValueHasStandoff: ReadPropertyInfoV2 = makeProperty(
```

Comment:
Currently we never serve standoff with a `TextValue`, but we send either XML or HTML (if an XSL transformation is attached to the mapping used) to the client. But we could surely serve the plain text and the standoff separately. 

We would just need to come up with a good idea how the client would request the format (as a segment in the URL or in the HTTP accept header).